packages/vue-cli-plugin-vuetify/generator/tools/vuetify.js

```diff
@@ -61,7 +61,7 @@ function renderFiles (api, { opts }) {
 }
 
 function addImports (api) {
-  api.injectImports(api.entryFile, "import vuetify from './plugins/vuetify';")
+  api.injectImports(api.entryFile, "import vuetify from './plugins/vuetify'")
```

Comment:
linter should be catching this and changing the double quotes to single.

Reply:
Do you want it like this `'import vuetify from \'./plugins/vuetify\''`

Reply:
disregard, I missed the inner quotations